src/MongoDB.Driver.Core/Core/Operations/RetryabilityHelper.cs

```diff
@@ -44,10 +42,7 @@ static RetryabilityHelper()
                 typeof(MongoNodeIsRecoveringException)
             };
 
-            __resumableChangeStreamExceptions = new HashSet<Type>(resumableAndRetryableExceptions)
-            {
-                typeof(MongoCursorNotFoundException)
-            };
+            __resumableChangeStreamExceptions = new HashSet<Type>(resumableAndRetryableExceptions);
```

Comment:
* MongoConnectionException => network error

* MongoNodeIsRecoveringException => The codes for this error MongoNodeIsRecoveringException correspond to the provided in the spec. But this error can also be generated for these error messages:
https://github.com/mongodb/mongo-csharp-driver/blob/master/src/MongoDB.Driver.Core/Core/Misc/ExceptionMapper.cs#L145-L146 This is not described in the spec. But I guess it's just a different way to specify codes from here: https://github.com/mongodb/mongo-csharp-driver/blob/master/src/MongoDB.Driver.Core/Core/Misc/ExceptionMapper.cs#L132-L136
* MongoNotPrimaryException (similar to above) => the codes for this error correspond to the spec. But this error can also be triggered for this error message https://github.com/mongodb/mongo-csharp-driver/blob/master/src/MongoDB.Driver.Core/Core/Misc/ExceptionMapper.cs#L150. This is not described in the spec. But I guess it's just a different way to specify the codes itself.

cc: @rstam

Reply:
OK I guess?

Reply:
I think so, I just wanted to have another opinion about it